### ISSUE TITLE
ANSI Invert displays incorrect colours

### ISSUE THREAD

**Issue by Starwort** [microsoft/terminal#6671]
# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd] Microsoft Windows [Version 10.0.18362.900]
Windows Terminal version (if applicable): 1.0.1401.0

Any other software? None relevant
```

# Steps to reproduce

Set your foreground and white colours, (as well as background and black colours) to something you will be able to distinguish.
For the purpose of this example, foreground is green, background is dark (#121212 for Terminal and #0c0c0c for WSL TTY), white (terminal) is red, and black (terminal) is blue
Launch e.g. Python (anything that can print ANSI codes), print ANSI invert and restore codes as follows:
```py
>>> print('\x1b[7m')
>>> print('\x1b[27m')
```

# Expected behaviour

Output similar to in the WSL dedicated terminal:
![image](https://user-images.githubusercontent.com/16487249/85644758-cc286780-b68f-11ea-9aae-58cf9fb05663.png)

# Actual behaviour

White and Black configured colours are being used in place of swapped foreground and background
In PowerShell:
![image](https://user-images.githubusercontent.com/16487249/85644810-e8c49f80-b68f-11ea-9f73-c8d2038eeaec.png)
In WSL:
![image](https://user-images.githubusercontent.com/16487249/85644829-fbd76f80-b68f-11ea-9724-dbb5ecaf9979.png)
In CMD:
![image](https://user-images.githubusercontent.com/16487249/85644864-10b40300-b690-11ea-95be-b0f13e1142ca.png)

**Comment by Starwort** [comment]
Now I can set my black/white back properly :eyes:

**Comment by j4james** [comment]
It looks like this is another variant of issue #2661. It'll be fixed by PR #6506.
